app/src/androidTest/java/com/example/concordia_campus_guide/activities/InfoCardVanierLibraryTest.java

```diff
@@ -54,7 +54,7 @@
             GrantPermissionRule.grant(
                     "android.permission.ACCESS_FINE_LOCATION",
                     "android.permission.ACCESS_COARSE_LOCATION",
-                    "android.permission.WRITE_CALENDAR",
+                    "android.permissigit on.WRITE_CALENDAR",
```

Comment:
wat

Reply:
I don't even know how this happened. I will fix them immediately